### DISCUSSION TITLE
Welcome to jquery-ui Discussions!

### DISCUSSION THREAD

**Discussion by mgol** [jquery/jquery-ui#1992] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lauren-mengert** [comment]
Hi @mgol! Question for you - Is there a way to track the date selected in your React app's state? I see that the UiDatepicker component tracks it, but I seem to be too much of noob to figure out how to access that data from another component. I tried adding an onChange handler to the UiDatepicker but it does absolutely nothing. This datepicker does everything I need it to do EXCEPT let me track the selected date in another component. I'm currently getting the value in my other component by using `document.getElementById`, but that's a bad practice for React! Any help for a noob would be appreciated

**Reply by mgol** [reply]
Would the option `onUpdateDatepicker` -  new in 1.13.0 - work for you? https://api.jqueryui.com/datepicker/#option-onUpdateDatepicker

**Comment by zidoxx** [comment]
Hi, please add the new jQuery UI 1.13.0 released to https://cdnjs.com/ if it's possible😃

**Reply by mgol** [reply]
This is outside of our control; you'll need to contact the project maintainers for that.
